### ISSUE TITLE
Bug: when using touchpad to zoom in and out on Uploaded image

### ISSUE THREAD

**Issue by Trivinyx** [TheAtlasRepository/ImgPDF2Map#106]
The touchpad zoom is to sensetive on image in splitview, on the image when zooming with touchpad the sense is too high.

**Comment by fredrikmoger** [comment]
Agree, a bug related to this is that it zooms on the center of the image view and not the cursor which is a bit confusing.
